test/logger.test.js

```diff
@@ -32,41 +31,155 @@ describe('strigo-node-logger', () => {
     });
 
     it('should not write debug printout when set up with info level', () => {
-      const log = setupNodeLogger({ env: 'prod', level: 'info' });
+      const log = setupNodeLogger({ json: true, level: 'info' });
```

Comment:
The `json: true` seems redundant here (i.e. unrelated to what's being tested)

Reply:
i've added it here since in the last comment you pointed out that changes to defaults might break tests, so I thought maybe it's just better to "set everything in stone" ?

Reply:
You should be explicit about what the test expects; in this case, it expects all defaults _except_ it explicitly states `with info level` and should therefore reflect that. This also helps in catching when changing defaults breaks existing behavior (either because the test is too strict, or because something actually broke).

The whole point is creating a situation where, with respect to test runs, _false negatives_ (test fails where it should succeed) are more likely than _false positives_ (test succeeds when it should fail), because the former is immediately visible to programmers whereas the latter creates a false sense of security.